app/controllers/search_organisation.js

```diff
@@ -1,6 +1,8 @@
 import Ember from "ember";
 
 export default Ember.Controller.extend({
+  queryParams: ["backToNewUser"],
+  backToNewUser: false,
```

Comment:
Can we use normal browser back behaviour, we've been trying to step away from these kinds of hacks.
If there's a reason we can't, then at least let's make it usable for any page with a `redirectTo` param which can support any page